library/systemd/src/lib/yast2/system_service.rb

```diff
@@ -239,7 +239,8 @@ def currently_active?
     def start_modes
       @start_modes = [:on_boot, :manual, :on_demand] unless found?
       return @start_modes if @start_modes
-      @start_modes = [:on_boot, :manual]
+      @start_modes = [:manual]
+      @start_modes << :on_boot unless service.static?
```

Comment:
does it make sense at all? If service is static, then it will always run like udev. so for me different should be on_boot and on_demand, but even for on_demand e.g. dbus will start automatic during system boot.

Reply:
Think, for instance, about tftpd.Perhaps you do not want that service to be started.

Reply:
well, documentation says that static services cannot be enabled neither disabled as it missing "Install" section. It means it can be run only as depency either for socket or by other services ( like udev ). So I am not sure if manual is right one, but neither on_boot sounds good for me. But for now we do not have anything better, so lets keep it with manual. But please ensure that we will never run enable/disable on it as it won't work.